agent/runner/actions/mongodb_explain_action.go

```diff
@@ -89,28 +88,13 @@ func (a *mongodbExplainAction) Run(ctx context.Context) ([]byte, error) {
 	}
 	defer client.Disconnect(ctx) //nolint:errcheck
 
-	var eq proto.ExampleQuery
-
-	err = bson.UnmarshalExtJSON([]byte(a.params.Query), true, &eq)
-	if err != nil {
-		return nil, errors.Wrapf(err, "Query: %s", a.params.Query)
+	if notMinMongoDBVersion(ctx, client) {
```

Comment:
let's drop this check, it will simplify code. No need for additional check, we can just write in documentation that we don't support EOLed versions.